client/additional-methods-setup/upe-preview-methods-selector/add-payment-methods-task.js

```diff
@@ -214,7 +214,6 @@ const AddPaymentMethodsTask = () => {
 
 	const prepareUpePaymentMethods = ( upeMethodIds ) => {
 		return upeMethodIds.map( ( key ) => {
-			// TODO : fix in https://github.com/Automattic/woocommerce-payments/issues/10182 to remove duplicated logic
```

Comment:
No longer needed - this was added because there was duplicated logic on the currencies set in the `paymentMethodsMap`. This PR (and the payment methods definitions) solve that issue, providing the currencies from the backend.